ga4gh/frontend.py

```diff
@@ -497,6 +497,12 @@ def searchDatasets():
         flask.request, app.backend.runSearchDatasets)
 
```

Comment:
Fronend.py seems to have acquired execute permissions. Please change this back. Ditto for _protocoldefinitions.py.